Marlin/src/HAL/STM32/tft/gt911.cpp

```diff
@@ -0,0 +1,208 @@
+/**
```

Comment:
this file should not go to `tft_io` folder?

Reply:
If… it is compatible with multiple platforms. This carries its own Software I2C class too. Maybe the code can be refactored to use some existing or shared IIC interface.

Reply:
At the moment, each HAL has its own tft files, in the folders `src/HAL/.../tft` so if these implementations share any common code, they might be reduced in size by moving some of their code to a `HAL/shared/tft` folder. But, these TFT files are not very large.